### ISSUE TITLE
Weighting Training Set

### ISSUE THREAD

**Issue by azhu529** [lululxvi/deepxde#535]
Opening a separate issue because the previous one was closed:

Hi @lululxvi,

Do you mind pointing out the relevant code files/changes that are needed to weight the training dataset with integers?

E.g. if I have points [x1, x2] and I want to assign weights [10,1], is there a more efficient method than explicitly writing out 11 points with repeats? That seems resource-intensive (the same goes for RAR); I'd like to use a smaller dataset with weights instead of denser sampling.

Thanks!

**Comment by lululxvi** [comment]
There are at least two ways:
1. Modify the loss:
https://github.com/lululxvi/deepxde/blob/3e6842069e0c377e7b0b80c3b9973bc8001927ae/deepxde/data/pde.py#L125
2. Modify the PDE residual by considering the weights
